### ISSUE TITLE
Mass placement of full blocks (whilst holding right click) sometimes places extra blocks.

### ISSUE THREAD

**Issue by schlid** [tsunamayo/Skywanderers#730]
As all the other blocks use shift to initiate mass placement, this is avoided, as you only place multiple blocks when you want to. 

However, as you don't have to "initiate" the action with full blocks, interesting things like this can happen pretty often, even when looking at the block center:

https://clips.twitch.tv/LaconicKathishDogeNotATK

Making it so pressing shift is necessary to mass-place for even full blocks would alleviate this issue.

**Comment by tsunamayo** [comment]
Yes I noticed that also, this is annoying I will fix it no pb. Thanks!
